workspaces/redhat-resource-optimization/plugins/redhat-resource-optimization/report.api.md

```diff
@@ -28,7 +29,7 @@ export const ResourceOptimizationIconOutlined: (
 ) => React_2.JSX.Element;
 
 // @public (undocumented)
-export const ResourceOptimizationPage: Router;
+export const ResourceOptimizationPage: Router_2;
```

Comment:
@jkilzi curious to know why we made this change, can you please share few details?

Reply:
The change was produced bt [this line](https://github.com/jkilzi/backstage-community-plugins/blob/18498603f75cafe1fcdfbafc88938137fafa4399/workspaces/redhat-resource-optimization/plugins/redhat-resource-optimization/src/plugin.ts#L72). This file is generated automatically, what intrigues me is why the import statement in line 9 wasn't there before but now the script added it 🤷‍♂️
Probably there was a version bump for the script that generates the docs in the main branch and we received this update after I rebased the next branch onto main.

Reply:
okay got it, thanks for sharing!